Makefile

```diff
@@ -106,15 +107,20 @@ build-container:
 $(MANPAGES): %: %.md
 	@sed -e 's/\((skopeo.*\.md)\)//' -e 's/\[\(skopeo.*\)\]/\1/' $<  | $(GOMD2MAN) -in /dev/stdin -out $@
 
-docs: $(MANPAGES)
+.PHONY: image.docs
+image.docs:
```

Comment:
This should probably have a dependency on `vendor.conf` so that it does not grow stale.

Reply:
added